.pre-commit-config.yaml

```diff
@@ -0,0 +1,18 @@
+repos:
+- repo: https://github.com/PyCQA/bandit
+  rev: 1.8.3
+  hooks:
+  - id: bandit
+    args: [-r, -c, .bandit.yml]
+- repo: https://github.com/psf/black.git
+  rev: 25.1.0
+  hooks:
+  - id: black
+- repo: https://github.com/PyCQA/flake8
+  rev: 7.2.0
+  hooks:
+  - id: flake8
+- repo: https://github.com/pycqa/isort
+  rev: 6.0.1
+  hooks:
+  - id: isort
```

Comment:
Oh, right. So, we have recently switched to ruff, which is both faster and, [properly configured](https://github.com/scrapy/scrapy/blob/b86f00327a3d113a97525f68d38f77529d090f30/pyproject.toml#L245-L303), can replace all these tools and more.